samples/InputBindingSamples/GetProductsStoreProcedureFromAppSetting.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Azure.WebJobs.Extensions.Sql.Samples.Common;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql.Samples.InputBindingSamples
+{
+
+    public static class GetProductsStoredProcedureFromAppSetting
+    {
+        [FunctionName("GetProductsStoredProcedureFromAppSetting")]
+        public static IActionResult Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = "getproductscost100-storedprocedure-appsetting")]
```

Comment:
Along with updating the app setting name I'd suggest we name the route something more generic like `getproductsbycost`.

The route really we should just be thinking of from the users point of view - what makes sense for someone calling this API. We have other ways like the function name to indicate to the developer looking at this sample what this sample is demonstrating.